test/Spago/Docs.purs

```diff
@@ -0,0 +1,29 @@
+module Test.Spago.Docs where
+
+import Test.Prelude
+
+import Node.FS.Aff as FSA
+import Node.Path as Path
+import Registry.Version as Version
+import Spago.Command.Init as Init
+import Spago.Core.Config as Config
+import Spago.FS as FS
+import Test.Spec (Spec)
+import Test.Spec as Spec
+import Test.Spec.Assertions as Assert
+
+spec :: Spec Unit
+spec = Spec.around withTempDir do
+  Spec.describe "docs" do
+
+    Spec.it "documents successfully with no flags" \{ spago } -> do
+      spago [ "init" ] >>= shouldBeSuccess
+      spago [ "docs" ] >>= shouldBeSuccess
```

Comment:
Could we check that there exist some html files in the place where we are expecting them?

Reply:
@CharlesTaylor7 actually I'll merge this, let's get to this in the next PR